dist/functions.setup-appliance.sh

```diff
@@ -238,33 +238,23 @@ function prepare_database_setup {
   RAILS_ENV=production bin/rails db:migrate:status > /dev/null 2>&1
 
   if [[ $? > 0 ]];then
-    echo "Initialize MySQL databases (first time only)"
-    DATADIR_FILE=$(grep datadir -rl /etc/my.cnf*)
-    echo " - reconfiguring datadir in $DATADIR_FILE"
-    sed -i -E '0,/(#\s*)?datadir/ s!#\s*datadir\s*=\s*/var/lib/mysql$!datadir = /srv/obs/MySQL!' $DATADIR_FILE
-    echo " - installing to new datadir"
-    mysql_install_db
-    echo " - changing ownership for new datadir"
-    chown mysql:mysql -R /srv/obs/MySQL
-    MYSQL_LOG=$(grep log-error /etc/my.cnf.d/*.cnf|perl -p -e 's/.*:log-error=(.*)/$1/')
-    if [ -n "$MYSQL_LOG" ];then
-      echo " - prepare log file $MYSQL_LOG"
-      LOG_DIR=`dirname $MYSQL_LOG`
-      if [ ! -d $LOG_DIR ];then
-        mkdir -p $LOG_DIR
-        chown mysql:mysql $LOG_DIR
+    if ! [[ "$(lsb_release -si)" =~ ^(Debian|Ubuntu)$ ]];then
```

Comment:
Please don't use `lsb_release` as it's deprecated and not universally available anymore. Use the content of `/etc/os-release` instead to identify members of a distribution family.